numpy/core/tests/test_numeric.py

```diff
@@ -140,6 +140,12 @@ def test_round(self):
         tgt = [1.6, 72.5, 6.4, 3.2]
         assert_equal(np.around(arr, decimals=1), tgt)
 
+        s = np.float64(1.)
+        assert_(isinstance(round(s), int))
+        assert_(isinstance(s.round(), np.generic))
+        assert_equal(s.round(), 1.)
+        assert_equal(round(s), 1)
+
```

Comment:
Worth a test that precision is not lost I think
```suggestion
        s = np.uint64(2**64-1)
        assert_equal(round(s), 2**64-1)
```